### PR TITLE
Update Calico and Canal to use calico node v2.6.7

### PR DESCRIPTION
**Pull request by tmjd**
- Canal updated cni to v1.11.2

### CODE REVIEW

File: upup/pkg/fi/cloudup/bootstrapchannelbuilder.go [hunk 1]
```diff
@@ -476,8 +476,8 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		key := "networking.projectcalico.org"
 		versions := map[string]string{
 			"pre-k8s-1.6": "2.4.1",
-			"k8s-1.6":     "2.6.6",
-			"k8s-1.7":     "2.6.6",
+			"k8s-1.6":     "2.6.8",
```
Comment (robinpercy) [inline upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: Thanks for the PR @tmjd. 

Should these versions not be 2.6.7?

Comment (tmjd) [reply upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: Probably, I had duplicated what was done with the `pre-k8s-1.6` as it has +1 on the minor.  I think maybe that one was bumped to 2.4.1 when it should have been 2.4.0.

Comment (tmjd) [reply upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: Would it be safe to change the `pre-k8s-1.6` to 2.4.0?  I'm not sure if that would break anything.

Comment (robinpercy) [reply upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: It looks like the minor +1 convention was specific to (https://github.com/kubernetes/kops/commit/148a65e1ad9041ceb1d46932fc6214d1a61a58e6). So we should definitely leave the pre-k8s-1.6 version as-is.

If I'm understanding the original intention correctly, the new versions should be 2.6.7-kops.1 for both canal and calico. (As a preemptive step for the latter). Then if future kops-specific patches are necessary, we'll just bump the suffix. 

@justinsb can you please confirm this is the versioning convention you had in mind in https://github.com/kubernetes/kops/commit/148a65e1ad9041ceb1d46932fc6214d1a61a58e6?

File: upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.8.yaml.template [hunk 2]
```diff
@@ -98,12 +98,15 @@ spec:
         # for cluster communication
         - effect: NoSchedule
           operator: Exists
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
```
Comment (blakebarnett) [inline upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.8.yaml.template]: Hmm, have you tested this behavior?  Does calico handle being killed like this gracefully, I imagine there's some cleanup on shutdown...

Comment (tmjd) [reply upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.8.yaml.template]: All the manifests (if not all, we missed updating them) on the Calico docs site have been updated to include `terminationGracePeriodSeconds: 0` so this being set has seen a decent amount of testing.
There is no cleanup I'm aware of on calico-node shutdown.

File: upup/pkg/fi/cloudup/bootstrapchannelbuilder.go [hunk 3]
```diff
@@ -532,7 +532,7 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		versions := map[string]string{
 			"pre-k8s-1.6": "2.4.2-kops.1",
 			"k8s-1.6":     "2.4.2-kops.1",
-			"k8s-1.8":     "2.6.3-kops.2",
+			"k8s-1.8":     "2.6.8-kops.2",
```
Comment (robinpercy) [inline upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: I believe hte version should be 2.6.7-kops.1 but would like to get @justinsb's confirmation.  If so, I suspect we should add the suffix to the calico addons above as well.

Comment (tmjd) [reply upup/pkg/fi/cloudup/bootstrapchannelbuilder.go]: Thanks for the confirmation. I've pushed an update fixing this.

**Comment by chrislovecnm** [comment]
ci is not happy, if you run make ci locally you will see the problem. A rebase may help!

**Comment by justinsb** [comment]
/ok-to-test

/lgtm

(Tried fixing the merge using github's online functionality)

**Pull request merged**